guides/support-and-on-call.md

```diff
@@ -159,3 +159,36 @@ Outside office hours, the expected response depends on the severity of the
 incident. We have a set of factors that determine the severity of an incident,
 and have included these in our
 [Service Level Agreement](https://contracts.dxw.com/service_level_agreement.md).
+
+### On call escalation
+
+While you are on out of hours on call there is also a 2nd line on call rota
+formed of senior members of the technical operations team and the CTO. OpsGenie
+will alert them if you do not acknowledge an alert within 5 minutes. This is to
+make sure that alerts are dealt with in a timely manner. You should not feel bad
+about missing an OpsGenie alert. This is one of the reasons why we have the
+second line rota.
+
+You can also use OpsGenie to escalate an alert to them if you are unsure what to
+do, lack access or need a hand in general. If it is a major incident you may
+want to escalate so that they can deal with client communications. If the person
+on the 2nd line rota does not respond feel free to contact the Head of Technical
+Operations or the CTO to help you. You should also feel free to contact other
```

Comment:
How do I contact those two people out of hours when I'm bleary and not thinking straight?

Reply:
My phone number is on my Slack profile, but the only thing that'll make it through do-not-disturb settings in the middle of the night are calls from one of the OpsGenie numbers. Might need to look at escalation routes within OpsGenie?

Reply:
it can almost certaihly be escalated somehow also you can use the directory in opsgenie to get our numbers.

Reply:
ive found a way which i can document.